test/pacmacs-image-test.el

```diff
@@ -95,3 +95,32 @@
         (let ((color (aref (aref expected-canvas-data y) x)))
           (should (equal color
                          (pacmacs--get-image-pixel canvas-image x y))))))))
+
+(ert-deftest pacmacs--read-im-enum-image-test ()
+  (let* ((image-filename "test-data/2x2.txt")
+         (image (pacmacs--read-im-enum-image image-filename))
+         (expected-pixels '((0 0 "#ff0000")
+                            (0 1 "#00ff00")
+                            (1 0 "#0000ff")
+                            (1 1 nil))))
+
+    (should (= 2 (pacmacs--image-width image)))
+    (should (= 2 (pacmacs--image-height image)))
+
+    (-all-p (-lambda ((x y color))
+              (should (equal color (pacmacs--get-image-pixel image x y))))
+            expected-pixels)))
+
+(ert-deftest pacmacs--parse-im-enum-header-test ()
+  (let* ((input-header "# ImageMagick pixel enumeration: 80,40,255,srgba")
+         (expected-result (list 80 40 255)))
+    (should (equal expected-result
+                   (pacmacs--parse-im-enum-header input-header)))
+    (should-error (pacmacs--parse-im-enum-header "khooy") :type 'error)))
+
+(ert-deftest pacmacs--parse-im-enum-pixel-test ()
+  (let ((input-pixel "60,16: (255,255,255,255)  #khooy")
```

Comment:
I think that's better to add spaces here to match the original format closer:
```
(  255,  255,  255,  255)
```

So the test will fail if we drop whitespace handling from our regexes somehow.